mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -18,7 +18,7 @@ pipeline {
                 sh '''
```

Comment:
A high level question, do you need any of those [lines](https://github.com/google/llvm-premerge-checks/blob/master/containers/base-debian/Dockerfile#L41-L48)?

Reply:
We don't need clang-10. Our default clang is clang-11.